pkg/docker/config/config.go

```diff
@@ -719,3 +751,20 @@ func normalizeRegistry(registry string) string {
 	}
 	return normalized
 }
+
+// isNamespaced returns true if the provided registry contains one or more
+// subpaths.
+func isNamespaced(registry string) bool {
+	stripped := stripScheme(registry)
+	return strings.ContainsRune(stripped, '/')
+}
+
+// validateIdentityRemappingPrefix verifies that the input registry does not
+// have any not allowed prefix.
+func validateIdentityRemappingPrefix(registry string) error {
```

Comment:
The name does not make sense here, we are not validating an “identity remapping prefix”. (And the input is not a `registry`).

Blocked on the name for the concept, sure.

Reply:
Yep, not it's just `validateKey`